src/vs/workbench/contrib/debug/browser/debugSession.ts

```diff
@@ -848,7 +848,17 @@ export class DebugSession implements IDebugSession {
 					// and the user opted in telemetry
 					if (this.raw.customTelemetryService && this.telemetryService.isOptedIn) {
 						// __GDPR__TODO__ We're sending events in the name of the debug extension and we can not ensure that those are declared correctly.
-						this.raw.customTelemetryService.publicLog(event.body.output, event.body.data);
+						let data = event.body.data;
```

Comment:
Does it make sense to move this out to a helper method which would live in `debugTelemetry.ts`?

Reply:
The DebugTelemetry class isn't very accessible here; that's used for the top-level of the session and send to the global(?) telemetry service, and for this bit of telemetry we're sending to a separate instance of the service created from the `aiKey` inside the debug extension's package.json.

So perhaps not a method, but I did pop this to its own helper function in debugUtils.

Reply:
Thanks. I like that!